### PR TITLE
Lint a few more packages

### PR DESCRIPTION
**Pull request by remi-stripe**
r? @brandur-stripe 
cc @stripe/api-libraries 

Only did a few packages. Minor but I don't think `go test ./...` is able to catch linter errors which makes it easy to miss later. I'm saying this because I added my packages to the whitelist and only saw errors when explicitly using `golint` and not when doing `go test ./...`

### CODE REVIEW

File: applepaydomain/client.go [hunk 1]
```diff
@@ -14,22 +14,24 @@ type Client struct {
 	Key string
 }
 
-// New POSTs new ApplePayDomains.
+// New creates a new ApplePayDomain.
```
Comment (brandur-stripe) [inline applepaydomain/client.go]: Minor, but I kind of think that we should try to use the "human style" form of "Apple Pay domain" in these comments (it's all intended for documentation). The function signature makes it obvious that a `stripe.ApplePayDomain` will be created.

File: balance/client.go [hunk 2]
```diff
@@ -66,7 +66,7 @@ type Iter struct {
 	*stripe.Iter
 }
 
-// Charge returns the most recent BalanceTransaction
+// BalanceTransaction returns the most recent BalanceTransaction
```
Comment (brandur-stripe) [inline balance/client.go]: And similarly "most recent balance transaction".

File: card/client.go [hunk 3]
```diff
@@ -15,12 +15,12 @@ type Client struct {
 	Key string
 }
 
-// New POSTs new cards either for a customer or recipient.
-// For more details see https://stripe.com/docs/api#create_card.
+// New creates a new card
```
Comment (brandur-stripe) [inline card/client.go]: Can you add punctuation on this one and the other `New`?

**Comment by remi-stripe** [comment]
> The default make target actually does a pretty good job of just running every known check for a slightly more thorough look to make sure everything is working.

Ohhhhh. I totally did not think I could just use `make` but it does work as expected so I'll know for next time.

**Comment by remi-stripe** [comment]
@brandur-stripe fixed the comments, thanks for catching all of those. PTAL!

**Comment by brandur-stripe** [comment]
LGTM. Thanks!

**Pull request merged**